topics/chaos-engineering/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: site-reliability-engineering
+display_name: Chaos Engineering
+related: sre, testing
+short_description: Chaos engineering is the process of testing a distributed computing system to ensure that it can withstand unexpected disruptions
```

Comment:
Perhaps this will work?
```suggestion
short_description: The process of testing distributed computing systems to ensure that they can withstand unexpected disruptions.
```